README.md

```diff
@@ -1,65 +1,283 @@
+
+## MinIO Operator and `kubectl` Plugin
+
+The MinIO Operator extends the Kubernetes API to support deploying MinIO-specific
+resources as a Tenant in a Kubernetes cluster.
+
+The MinIO `kubectl minio` plugin wraps the Operator to provide a simplified interface
+for deploying and managing MinIO Tenants in a Kubernetes cluster through the
+`kubectl` command line tool.
+
+# Create a MinIO Tenant
+
+This procedure creates a 4-node MinIO Tenant suitable for evaluation and 
+early development using MinIO for object storage.
+
+- MinIO requires Kubernetes version 1.17.0 or later.
+
+- This procedure assumes the cluster contains a 
+  [namespace](https://github.com/minio/operator/blob/master/README.md#create-a-namespace) for
+  the MinIO Tenant.
+
+- This procedure assumes the cluster contains a
+  [`StorageClass`](https://github.com/minio/operator/blob/master/README.md#default-storage-class)
+  for the MinIO Tenant Persistent Volumes  (`PV`). The `StorageClass`
+  *must* have `volumeBindingMode: WaitForFirstConsumer`
+
+## 1) Install the MinIO Operator
+
+### Install using `kubectl krew`
+
+Run the following command to install the MinIO Operator and Plugin using `krew`:
+
+```sh
+   kubectl krew update
+   kubectl krew install minio
+```
+
+Run the following command to initialize the Operator:
 
 ```sh
 kubectl minio init
+
 ```
 
-Once the MinIO Operator is created, proceed with Tenant creation.
+## 2) Create a New Tenant
 
-## Tenant Setup
+The following `kubectl minio` command creates a MinIO Tenant with 4 nodes, 16
+volumes, and a total capacity  of 16Ti. This configuration requires
+*at least* 16 
+[Persistent Volumes](https://github.com/minio/operator#Local-Persistent-Volumes).
 
-A Tenant is a MinIO cluster created and managed by the Operator. Before creating tenant, please ensure you have requisite nodes and drives in place and relevant PVs are created.
+```sh
+   kubectl minio tenant create minio-tenant-1 \
+      --servers 4                             \
+      --volumes 16                            \
+      --capacity 16Ti                         \
+      --namespace minio-tenant-1              \
+      --storageClassName local-storage        \
+```
 
-In below example, we ask MinIO Operator to create a Tenant yaml with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVs of 1Ti each per node, with a total of 4 nodes, before attempting to create the MinIO tenant.
+- The `minio-tenant-1` argument specifies the name of the MinIO Tenant. The MinIO
+  Operator uses this name as a prefix for certain resources in the Tenant.
 
-Since MinIO has built-in erasure-code support for high-availability and data protection, we recommend the following CSI drivers for high-performance and scalability:
+- The `--servers` field indicates the number of `minio` pods to deploy into the cluster.
+  The cluster *must* have at least one available Node per `minio` pod. 
```

Comment:
Do we need to be pedantic and add "worker node" here?

Reply:
I think it's worth being that specific. The Kubernetes docs uses the two terms interchangeably, but might as well avoid that potential edge.